granulate_utils/linux/elf.py

```diff
@@ -71,18 +71,19 @@ def get_elf_arch(elf: ELFType) -> str:
         return elf.get_machine_arch()
 
 
-def get_elf_buildid(elf: ELFType) -> Optional[str]:
+def get_elf_buildid(elf: ELFType, section: str, note_check: Callable[[Any], Any] = lambda n: True) -> Optional[str]:
```

Comment:
Does it make sense to have a default value for the note? And furthermore, a default value that behaves differently that the past behavior?

I'd just remove the default and require the caller to pass it.

If you can add types that's also good :)

Reply:
👍